server/templates/staff/course/students.list.html

```diff
@@ -37,6 +38,11 @@ <h3 class="box-title"><span> {{ title }} </span></h3>
                     <td class="role"><span class="label label-success">{{item.role}}</span></td>
                     <td class="timestamp" data-timestamp="{{ item.created }}">{{ utils.local_time(item.created, current_course) }}</td>
                     <td class="section">{{item.section}}</td>
+                    <td class="unenroll">
+                        {% call forms.render_form(unenroll_form, action_url=url_for('.unenrollment', cid=current_course.id, user_id=item.user.id), action_text='Unenroll',
```

Comment:
a confirm dialog would be nice. It'd be nice to standardize on a way to get a form to have a confirm dialog - since we could use it multiple places. We might want to leave it out of the scope of this PR - (@Rohit-L - if you want to add this - feel free to) 

On the student side - we do this with a jQuery function. Something similar for the admin panel but using data attributes on the form would be neat. 

https://github.com/Cal-CS-61A-Staff/ok/blob/eafb805028c9ccb5812fdd9c6ce89e784cf6d5ba/server/static/js/comments.js#L13

`<button type="submit" class="btn form-confirm" data-confirm="Are you sure you want to delete student@example.com?">`